### ISSUE TITLE
Autoupdate applications

### ISSUE THREAD

**Issue by antmordel** [fvarrui/JavaPackager#215]
I'm submitting a…

- [ ] bug report
- [x] feature request
- [ ] other

**Short description of the issue/suggestion:**
Hello @fvarrui - I love your work and I've been following this project for a while. It works as intended and it's easy to use. I have now a doubt that is: I would love to have an autoupdater in my apps. Meaning my application will connect to a URL pointing to a server to check if there are new versions, and if so, will download the new version, close the current app, and start the new installer (or replace the .exe file with the new version).

How do you think I should proceed on that? Surely this has come up before. 

Many thanks and have a happy weekend!

**Comment by fvarrui** [comment]
Hi @antmordel!
Have you had a look at [update4j](https://github.com/update4j/update4j)?

**Comment by fvarrui** [comment]
It's useful to replace specific installed files with other ones (e.g. several files downloaded from a server)

**Comment by fvarrui** [comment]
Yes, **update4j** is a cross-platform autoupdate solution ... I haven't used it before, just read about it. But keep in mind that it has nothing to do with JavaPackager. It has to be implemented as part of your app.

**Comment by fvarrui** [comment]
Hi @antmordel! 
Can I close this issue? I'm catching up on JavaPackager after several busy months 😅
